api/src/OpenTelemetry/Trace/Id.hs

```diff
@@ -44,7 +45,7 @@ module OpenTelemetry.Trace.Id (
   traceIdBaseEncodedText,
 
   -- * Working with 'SpanId's
-  SpanId,
```

Comment:
I don't really want to export `TraceId` / `SpanId` constructors for public consumption in general, but I'd be okay with exporting it from an `Internal` module.

Reply:
I see.

How about this change? https://github.com/iand675/hs-opentelemetry/pull/49/commits/da6e4adf486139996f42761920eac42b16ccb59a

- move implementations to `OpenTelemetry.Internal.Trace.Id`
- control exposure at `OpenTelemetry.Trace.Id`